r/R/dplyr.R

```diff
@@ -265,6 +267,8 @@ filter_mask <- function(.data) {
   # Then add the column references
   # Renaming is handled automatically by the named list
   env_bind(f_env, !!!lapply(.data$selected_columns, var_binder))
+  # Then bind the data pronoun
+  env_bind(f_env, .data = data_pronoun)
```

Comment:
What about this (and delete the data_pronoun defs above)?

```suggestion
  data_pronoun <- lapply(.data$selected_columns, var_binder)
  env_bind(f_env, !!!data_pronoun)
  # Then bind the data pronoun
  env_bind(f_env, .data = data_pronoun)
```

I think L261 will miss the column renaming but this will pick it up (and should be marginally more efficient)